Scripts/BuildEngine.ps1

```diff
@@ -68,6 +68,10 @@ else {
     }
 }
 
+# Add NuGet to the PATH for the current session
+$installPath = Split-Path -Path $nugetProgram -Parent
+$env:PATH = "$installPath;$env:PATH"
```

Comment:
is doing this won't continually concatenate the Nuget Path every run? let's say I run the script 3 times

Reply:
okay, we can check if its already there before adding it to path